client/src/Components/CommonComponent/Header/style.css

```diff
@@ -0,0 +1,16 @@
+.header i{
```

Comment:
also, use BEM the class name like that `header__icon`  and write this commit on the above of file to know any one use this file `/* when pass Icon put insilde it <Icon type="..." className="header__icon" /> */`

Reply:
ok idit it thanks shorouq :+1: